bin/wmagent-resource-control

```diff
@@ -167,13 +167,12 @@ def printThresholds(resourceControl, desiredSiteName):
         print "%s - %d running, %d pending, %d running slots total, %d pending slots total%s:" % (
                         siteName, runningJobs, pendingJobs, runningSlots, pendingSlots, stateMsg)
 
-        for task in siteThresholds['thresholds']:
+        for task, thres in siteThresholds['thresholds'].iteritems():
```

Comment:
Right, we have to do a global converter run, so we should program for the functionality we want and look forward to the glorious future where the right functionality also looks the prettiest. :-)